pipeline_steps/common.groovy

```diff
@@ -396,5 +396,20 @@ def docker_cache_workaround(){
    sh "touch -t 201704100000 *.txt"
 }
 
+def is_doc_update_pr(git_dir) {
+  if (env.ghprbPullId != null) {
+    dir(git_dir) {
+      def output = sh(script: """#!/bin/bash
+      git show --stat=400,400 | awk '/\\|/{print \$1}' \
+        | egrep -v -e '.*md\$' -e '.*rst\$' -e '^releasenotes/' \
+        || echo "Skipping build as no non-doc changes were detected"
+      """, returnStdout: true)
+      print output
+      return output.contains("Skip")
```

Comment:
If a path includes the characters "Skip" it looks like this could cause a PR to be skipped by mistake. I think it would be better to have a defined output from the script.